src/Components/Web.JS/src/Platform/Circuits/CircuitStartOptions.ts

```diff
@@ -22,19 +22,22 @@ export interface ReconnectionOptions {
   maxRetries: number;
   retryIntervalMilliseconds: number;
   dialogId: string;
+  reloadOnCircuitRejected: boolean;
 }
 
 export interface ReconnectionHandler {
   onConnectionDown(options: ReconnectionOptions, error?: Error): void;
   onConnectionUp(): void;
+  onConnectionRejected(options: ReconnectionOptions): void;
 }
 
 const defaultOptions: CircuitStartOptions = {
     configureSignalR: (_) => { },
     logLevel: LogLevel.Warning,
     reconnectionOptions: {
       maxRetries: 8,
-      retryIntervalMilliseconds: 20000,
+      retryIntervalMilliseconds: 3000,
```

Comment:
I'm not sure we'd want to make such an aggressive change by default. Developers who prefer a 3s interval can already configure that. For everyone else, this means that clients will be attempting reconnection very rapidly which might result in a weird UI if they have customized it.

Reply:
In other words the final decision is: I'll revert it, am I correct?

Reply:
Reverted